src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -194,11 +209,9 @@
           DependsOnTargets="CreateAzureStorage"
           Condition="'$(Performance)' != 'true' or '$(FuncTestsDisabled)' != 'true'">
     <!-- create item group of functional tests -->
-    <ItemGroup>
-      <FunctionalTest Include="$(TestArchivesRoot)**/*.zip" />
-      <FunctionalTest Include="$(AnyOSTestArchivesRoot)**/*.zip" />
-      <FunctionalTest Condition="'$(TargetsUnix)' == 'true'" Include="$(UnixTestArchivesRoot)**/*.zip" />
-    </ItemGroup>
+    <CreateItem Include="$(ForUploadList)">
+      <Output TaskParameter="Include" ItemName="FunctionalTest"/>
+    </CreateItem>
```

Comment:
why we didn't do 

<ItemGroup>
  <FunctionalTest Include="$(ForUploadList)" />
</ItemGroup>